src/components/ProfileNavigation/profileNavigation.styles.ts

```diff
@@ -0,0 +1,43 @@
+import { styled } from '@mui/system';
+import { Drawer, List, Toolbar } from '@mui/material';
+
+export const CustomToolbar = styled(Toolbar)(() => ({
+  justifyContent: 'flex-end',
+}));
+
+export const CustomDrawer = styled(Drawer)(({ theme }) => ({
+  [theme.breakpoints.down('md')]: {
+    '.MuiDrawer-paper': {
+      width: '100%',
+    },
+  },
+
+  '.MuiDrawer-paper': {
+    border: 0,
+  },
+}));
+
+export const CustomList = styled(List)(({ theme }) => ({
+  [theme.breakpoints.up('md')]: {
+    paddingTop: 70,
+  },
+
+  '& > a': {
+    color: theme.palette.text.primary,
+    [theme.breakpoints.up('md')]: {
+      width: '100%',
+      padding: '8px 40px',
+      '&:not(:last-of-type)': {
+        marginBottom: 10,
```

Comment:
I check Figma's layout and there is some bug that can be improved. list items have their padding, so that margin is unnecessary.